### ISSUE TITLE
Hooks DCS-gRPC.lua - file not closed after reading

### ISSUE THREAD

**Issue by Applevangelist** [DCS-gRPC/rust-server#237]
Please amend the code like so:

```
  -- load settings from `Saved Games/DCS/Config/dcs-grpc.lua`
  do
    log.write("[GRPC-Hook]", log.INFO, "Checking optional config at `Config/dcs-grpc.lua` ...")
    local file, err = io.open(lfs.writedir() .. [[Config\dcs-grpc.lua]], "r")
    if file then
      local f = assert(loadstring(file:read("*all")))
      setfenv(f, GRPC)
      f()
	  io.close(file) -- close the file again or it stays locked
      log.write("[GRPC-Hook]", log.INFO, "`Config/dcs-grpc.lua` successfully read")
    else
      log.write("[GRPC-Hook]", log.INFO, "`Config/dcs-grpc.lua` not found (" .. tostring(err) .. ")")
    end
  end
  ```

**Comment by Applevangelist** [comment]
#239

**Comment by rurounijones** [comment]
Fixed by #239
